Core/LICENSE

```diff
@@ -0,0 +1,11 @@
+Owner: FlowCrypt Limited
```

Comment:
Please remove the `LICENSE` file, it will go by the main `LICENSE` file in the repo.